Automattic/wp-calypso
# Reader: Move to its own entry point

This PR is an experiment to move the Reader to its own entry point. Right now it doesn't do a wonderful job of reusing all the code possible between the different boot sequences, but I think this can be done in subsequent PRs if we decide to go with this approach.

#### Changes proposed in this Pull Request

* Create own boot for Reader
* Create a new entry point for the Reader
* Move out all Reader sections to the new entry point
* State: Update redux bridge when a reducer is added
* Layout: Remove link for notifications masterbar item

#### Testing instructions

* Checkout this branch and build Calypso from scratch.
* Smoke test all of Reader and verify it works well and as before.
* Verify the root route `/` works as it did before.
